src/backend/web/app.js

```diff
@@ -19,6 +24,15 @@ app.use(logger);
 
 app.use(cors());
 
+const server = new ApolloServer({
+  typeDefs,
+  resolvers,
+  context: { storage },
```

Comment:
Question: why not use node's module resolution for this instead of passing this down via `context`?  We can just `requrie()` storage where you need it when defining the queries and use it directly.

Reply:
Initially that's what I tried (`require()` in the `.js` file where the queries are), but it seems `Apollo` doesn't allow access to the queries from _outside_ the server, so the only way that it'd let me use storage functions _in_ the queries was by passing them down via `context`.

Reply:
Fascinating.  How did it disallow it?  The `storage` via `require()` wasn't defined?

Reply:
This is going to (potentially) complicated a refactor I wanted to make, moving data logic into classes like `Post` and `Feed` and slowly getting rid of `storage` completely.  Maybe we would just pass both `Post` and `Feed` down on `context`, and it would be OK.

Reply:
@humphd I tested it again and it turned out it's ok to require storage where the queries are. Maybe I got some sort of syntax error when I was testing it and I interpreted as Apollo not being happy about with what I was doing. Thank you for asking about this, without your question I'd definitely have missed this.